update_prevalence.py

```diff
@@ -1352,38 +1368,48 @@ def parse_romania_prevalence_data(cache: DataCache, data: AllData) -> None:
 def parse_canada_prevalence_data(cache: DataCache, data: AllData) -> None:
     populate_since = canada_effective_date - timedelta(days=16)
     canada_one_week_ago = canada_effective_date - timedelta(days=6)
+
     try:
-        canada_regions = parse_json(cache, CanadaOpenCovidRegions, CanadaOpenCovidRegions.SOURCE)
+        # pull lists of health regions, provinces and territories and
+        # their abbreviations
+
+        canada_regions = parse_csv(cache, CovidTimelineCanadaRegion, CovidTimelineCanadaRegion.SOURCE)
+
+        canada_provinces = parse_csv(
+            cache, CovidTimelineCanadaProvinceOrTerritory, CovidTimelineCanadaProvinceOrTerritory.SOURCE
+        )
     except pydantic.error_wrappers.ValidationError as e:
         print_and_log_to_sentry(f"Discarding county-level data from Canada due to error: {e}")
         return
```

Comment:
I would like to do more here to handle more potential API issues - e.g. right now the only cases that would fail would be the CSVs above going away or columns being renamed.  There are multiple additional API calls below this which are currently unhandled.  I think it would confuse the diff more than is worth it for this PR, however, and there are a couple of different approaches to think through, so I'd propose to punt that for now.